docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -137,21 +93,14 @@ To start a validator, run the following commands from a Linux terminal:
 .. code-block:: console
 
    $ sawtooth keygen --key-dir /etc/sawtooth/keys/ validator
-   $ validator -vv --endpoint tcp://localhost:8800
+   $ validator -vv --endpoint tcp://127.0.0.1:8800 \
+     --bind network:tcp://127.0.0.1:8800 \
+     --bind component:tcp://127.0.0.1:4004
```

Comment:
Well, the initial reason was that the default in the stable debs was out of sync with the default in the stable docs somehow, so I wanted to add some redundancy in case we change the port again. I think setting this explicitly also makes it clearer where the endpoint used by the REST API and transaction processor comes from, and that they are components.